app/components/Header.js

```diff
@@ -64,9 +64,10 @@ const Header = () => {
                 </a>
                 <a
                   style={{marginLeft: '20px'}}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  href="http://localhost:3000/question/66"
+                  // (Dylan) Why these next 2 lines?
+                  // target="_blank"
+                  // rel="noopener noreferrer"
+                  href="/admin"
```

Comment:
...and now that this goes to Admin, I don't see why you'd open a new window anyway. Looks good :)